validations/src/validations.rs

```diff
@@ -1171,6 +1197,11 @@ pub fn validate_tally_transaction<'a>(
         .into());
     }
 
+    #[cfg(test)]
+    println!(
+        "expected_ta_tx.tally: {}",
+        hex::encode(&expected_ta_tx.tally)
+    );
```

Comment:
This should be removed

Reply:
why?

Reply:
I added it to help me write the tests, if the tests are done we don't need it.

Reply:
I had to modify several times the rng tests, so I think it could be useful in the future too